src/main/java/frc/robot/poseestimator/PoseEstimatorMath.java

```diff
@@ -0,0 +1,94 @@
+package frc.robot.poseestimator;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Twist2d;
+import frc.robot.poseestimator.observations.VisionObservation;
+
+public class PoseEstimatorMath {
+
+	public static Twist2d addGyroToTwist(Twist2d twist, Rotation2d currentGyroAngle, Rotation2d lastGyroAngle) {
```

Comment:
From the name, it sounds like it preforms stuff in place -- but it doesn't change the twist.